### ISSUE TITLE
airsonic.lobs file has grown excessively

### ISSUE THREAD

**Issue by Dikakus** [airsonic-advanced/airsonic-advanced#912]
## Problem description

I don't know exactly why but the airsonic.lobs file has grown excessively, reaching 1,4 G when the usual was 576 K.

Would it be a way to reduce its size? I tried reloading folders and cleaning database with the web application but the size is the same.

### Steps to reproduce

I don't know what the steps are to reproduce the issue, but a ls command shows:

```
total 1,4G
drwxrwxr-x 1 airsonic airsonic  166 2022-05-27 12:35 .
drwxrwxr-x 1 airsonic airsonic  302 2022-05-27 12:36 ..
-rw-r--r-- 1 airsonic airsonic   16 2022-05-27 12:47 airsonic.lck
-rw-rw-r-- 1 airsonic airsonic 1,4G 2022-05-27 12:41 airsonic.lobs
-rw-r--r-- 1 airsonic airsonic 4,7K 2022-05-27 12:41 airsonic.log
-rw-r--r-- 1 airsonic airsonic  104 2022-05-27 12:35 airsonic.properties
-rw-r--r-- 1 airsonic airsonic  52M 2022-05-27 12:32 airsonic.script
drwxr-xr-x 1 airsonic airsonic    0 2022-05-27 12:35 airsonic.tmp

```

When a previous copy of the data base shows:

```
total 47M
drwxr-xr-x 1 airsonic airsonic   94 2022-04-18 10:42 .
drwxrwxr-x 1 airsonic airsonic  302 2022-05-27 12:36 ..
-rw-rw-r-- 1 airsonic airsonic 576K 2022-04-15 20:17 airsonic.lobs
-rw-r--r-- 1 airsonic airsonic  103 2022-04-18 10:42 airsonic.properties
-rw-r--r-- 1 airsonic airsonic  47M 2022-04-18 10:42 airsonic.script

```

## System information

- Airsonic version: 11.0.0-SNAPSHOT.20220404145227 (2022/04/04)
- Operating system: Raspbian GNU/Linux bullseye, aarch64
- Java version: 11.0.14+9-post-Raspbian-1deb11u1
- Proxy server: None
- Client: Opera
- Language: Spanish

## Additional notes

*Include any extra notes here. Otherwise you may remove this section.*

**Comment by Dikakus** [comment]
> My lobs file is over 4G:
> 

So, do you think is it a normal behavior?

**Comment by randomnicode** [comment]
shutdown script didn't help?

**Comment by khers** [comment]
No, as I mentioned in @730, the solution of @729 got me started again but it did not address the large lobs file

**Comment by Dikakus** [comment]
> shutdown script didn't help?

Neither for me. :(

**Comment by eharris** [comment]
I've had this problem as well.  I'm not sure, but I think it might be related to podcasts that have large descriptions or other large attributes, as after I decided not to try to use podcasts through the app anymore and did a database reset and didn't setup podcasts again, the lobs did not come back.

**Comment by Dikakus** [comment]
Thank you for your comment @eharris. I prefer to keep all the functionalities in one single place, so I',m managing with that large file. On the other hand, I think "not use that" is not a ideal solution, although there may not be any possible solution due to the characteristics of the BD.
